app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java

```diff
@@ -0,0 +1,48 @@
+package io.github.droidkaigi.confsched2017.view.activity;
+
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import io.github.droidkaigi.confsched2017.R;
+import io.github.droidkaigi.confsched2017.databinding.ActivitySplashBinding;
+
+public class SplashActivity extends AppCompatActivity {
+
+    private static final String TAG = SplashActivity.class.getSimpleName();
+
+    private ActivitySplashBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = DataBindingUtil.setContentView(this, R.layout.activity_splash);
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            findViewById(android.R.id.content).setSystemUiVisibility(
+                    View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN | View.SYSTEM_UI_FLAG_LAYOUT_STABLE);
+        }
+
+        final ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+        scheduledExecutorService.scheduleAtFixedRate(
+                () -> binding.particleAnimationView.postInvalidate(),
+                0L,
+                40L,
+                TimeUnit.MILLISECONDS);
+
+        new Handler().postDelayed(() -> {
+            scheduledExecutorService.shutdown();
+            startActivity(new Intent(SplashActivity.this, MainActivity.class));
```

Comment:
In addition to that, what if user press a back button during splash screen? I recommend to assign this `Runnable` to a field and call `Hander#removeCallbacks(runnable)` on `Activity#onStop()`.